.github/workflows/build_makefile_test.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Test
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - tag: intel
+          build_args: "CC=mpiicpc ELPA_LIB_DIR=/usr/local/lib ELPA_INCLUDE_DIR=/usr/local/include CEREAL_DIR=/usr/include/cereal OPENMP=ON"
```

Comment:
Shall we use `CXX` here?

Reply:
Since v3.0.0 the compiler name in Makefile has been CC. I think either is OK, but changing it to CXX might require discussion with @Qianruipku and @mohanchen .

Reply:
> Since v3.0.0 the compiler name in Makefile has been CC. I think either is OK, but changing it to CXX might require discussion with @Qianruipku and @mohanchen .

I don't think using CC for cpp files is adequate. Please see https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html

Reply:
I understand what you mean. CC should be short for 'C compiler'. Let me ask them if they agree.